docs/querying/sql.md

```diff
@@ -334,6 +332,22 @@ simplest way to write literal timestamps in other time zones is to use TIME_PARS
 |<code>timestamp_expr { + &#124; - } <interval_expr><code>|Add or subtract an amount of time from a timestamp. interval_expr can include interval literals like `INTERVAL '2' HOUR`, and may include interval arithmetic as well. This operator treats days as uniformly 86400 seconds long, and does not take into account daylight savings time. To account for daylight savings time, use TIME_SHIFT instead.|
 
 
+### Reduction functions
+
+Reduction functions operate on zero or more expressions and return a single expression. If no expressions are passed
+as arguments, then the result is `NULL`. The expressions must all be convertible to a
+common data type, which will be the type of the result:
+*  If any argument is `NULL`, the result is `NULL`.
+*  If the arguments comprise a mix of numbers and strings, the arguments are interpreted as strings.
```

Comment:
is this implicit cast something that other databases do as well ? how about enforce an explicit cast ?

Reply:
The references that I looked at (added to the description) do implicit casting. The docs I added here are inspired by the ones for MySQL: https://dev.mysql.com/doc/refman/8.0/en/comparison-operators.html#function_least

Reply:
Thanks for pointer, do you see this applying to every reduce function ?

Reply:
I'm not sure what other reduction functions we'll want to add in the future, but ideally the behavior is consistent. That being said, I wouldn't be surprised if the docs need to be reorganized to accommodate future reduction functions.

Reply:
If am reading the code correctly this is applying to every reduce function.

Reply:
Yeah, I meant to say the docs and implementation may need to change later depending on whether/what other reduction functions are added in the future.